fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/t5_tokenizer_AA_special.json

```diff
@@ -2657,6 +2657,42 @@
       "rstrip": false,
       "normalized": false,
       "special": true
+    },
+    {
+      "id": 295,
+      "content": "<ALTERNATIVE>",
```

Comment:
in the bmfm_bio PR I asked if you could add "MUTATED" as a modifier special token. so if you did, it should affect this PR